src/NzbDrone.Core/HealthCheck/Checks/SlackUrlCheck.cs

```diff
@@ -0,0 +1,33 @@
+using System.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Localization;
+using NzbDrone.Core.Notifications;
+using NzbDrone.Core.Notifications.Slack;
+using NzbDrone.Core.ThingiProvider.Events;
+
+namespace NzbDrone.Core.HealthCheck.Checks
+{
+    [CheckOn(typeof(ProviderUpdatedEvent<INotificationFactory>))]
+    [CheckOn(typeof(ProviderDeletedEvent<INotificationFactory>))]
+    [CheckOn(typeof(ProviderStatusChangedEvent<INotificationFactory>))]
+    public class SlackUrlCheck : HealthCheckBase
+    {
+        private readonly INotificationFactory _notificationFactory;
+
+        public SlackUrlCheck(INotificationFactory notificationFactory, ILocalizationService localizationService)
+            : base(localizationService)
+        {
+            _notificationFactory = notificationFactory;
+        }
+
+        public override HealthCheck Check()
+        {
+            if (_notificationFactory.GetAvailableProviders().Where(n => n.Name.Equals("Slack")).Any(s => (s.Definition.Settings as SlackSettings).WebHookUrl.Contains("discord")))
```

Comment:
Should this be Where `n.Implementation` equals Slack?

Reply:
Very possibly yes 😂😂

Reply:
Moved it to `n.configcontract.equals("slacksettings")`